examples/ESA/ESA_WorldCereal_2021_MODELS_v100_preview.js

```diff
@@ -0,0 +1,59 @@
+// var dataset = ee.ImageCollection('ESA/WorldCereal/2021/MODELS/v100')
+var dataset = ee.ImageCollection('projects/worldcereal/assets/2021_v100')
+
+Map.setOptions('SATELLITE');
+
+// Width and height of the thumbnail image.
+var pixels = 1024;
```

Comment:
Not sure what the maximum is for your thumbnails, but if we would go for a global overview then setting this too low gives a very low-res image.

Reply:
Our standard is 256px (sorry, this is not documented yet)